src/control/drpc/modules.go

```diff
@@ -214,6 +215,8 @@ const (
 	MethodPoolGetACL MgmtMethod = C.DRPC_METHOD_MGMT_POOL_GET_ACL
 	// MethodListPools is a ModuleMgmt method
 	MethodListPools MgmtMethod = C.DRPC_METHOD_MGMT_LIST_POOLS
+	// MethodClusterEvent is a ModuleMgmt method
+	MethodClusterEvent MgmtMethod = C.DRPC_METHOD_MGMT_CLUSTER_EVENT
```

Comment:
I don't think this is a mgmt method, is it? It's always an upcall from the ioserver to the control plane, right? If that's the case, then I think this should be defined alongside the `_SRV_` methods, e.g. `DPRC_METHOD_SRV_NOTIFY_RAS`.

Reply:
ok, will separate mgmt cluster event and srv notify ras